cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -159,4 +159,9 @@
    * artifact fetching
    */
   public static final String PEER_NAMESPACE = "peerNamespace";
+
+  /**
+   * Option for URI to additional resources needed for program run. This is needed for tethered programs
+   */
+  public static final String PROGRAM_RESOURCE_URI = "programResourceUri";
```

Comment:
nit: "Option for URI to additional resources needed for program run" -> missing a word?

Reply:
reworded to sound less awkward